src/nvim/macros.h

```diff
@@ -148,6 +148,11 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
+// Windows defines a RGB macro that produces 0x00bbggrr color values for use
+// with GDI. Our macro is different, and we don't use GDI.
+#if defined(RGB)
```

Comment:
should probably check WIN32 also.

    #if defined(WIN32) && defined(RGB)

Reply:
I keep point not need check windows

Reply:
If RGB macro is defined by a library on non-Windows, we want to know about it, not silently override it. If that happens we should rename the macro.

Reply:
Done.